exercises/chapter11/test/Main.purs

```diff
@@ -129,5 +144,37 @@ This line should have been automatically deleted by resetSolutions.sh. See Chapt
         test "should fail if first is not a or b" do
           Assert.equal (Left ["Could not parse","Could not parse"])
             $ runParser asOrBs "foobar"
+
+    suite "Exercises Group - The RWS Monad" do
+      let 
+        runGame :: Game Unit -> RWSResult GameState Unit (List String)
+        runGame testGame = runRWS testGame env initialGameState
+        env = GameEnvironment { debugMode: false, playerName: "Phil" }
+
+        expectedCheaterState = GameState 
+          { inventory: S.fromFoldable [Candle, Matches] 
+          , items: M.empty
+          , player: Coords { x: 0, y: 0 } 
```

Comment:
I think it would be more correct to write a `playerHasAllItems` function that gets called below, rather than checking for game state equality and requiring the player to be at the origin coordinates.